modules/chapter2/pages/rbac.adoc

```diff
@@ -0,0 +1,138 @@
+# Role Based Access Control (RBAC)
+:navtitle: Role Based Access Control
+
+In the previous section, you learnt how to _authenticate_ users (that is, *who* they are). In this section you will learn how to _authorize_ users to perform actions in RHDH (that is, *what* they can do in RHDH).
+
+Role-Based Access Control (RBAC) is a security concept that controls access to resources in a system, and specifies a mapping between _users_ of the system, and the _actions_ they can perform on _resources_ in the system. You define _roles_ with specific _permissions_ , and then assign the roles to users.
+
+RBAC on RHDH is built on top of the *Permissions* framework in Backstage, which defines RBAC policies in code. Rather than define policies in code, the RHDH RBAC feature allows you to define policies in a declarative fashion using a simple CSV based format.
+
+To apply RBAC in RHDH, you need to do two things:
+
+. Configure an authenticated user as a *Policy Administrator*
+. Configure policies in a CSV file and import it into RHDH
+
+## Lab: Configure Policy Administrators
+
+NOTE: The main reason to declare policy administrators is to allow a certain limited number of authenticated users to invoke the RBAC REST API. The actual policies are defined in a separate CSV file and referenced in the `app-config-rhdh` ConfigMap. This can be done by OpenShift platform or cluster administrators who have access to the namespace where RHDH is running.
+
+. To declare users as policy administrators, add the following `permission` attribute to the `app-config-rhdh` ConfigMap (At the same indentation level as the *app* and below the *catalog* attribute). Ensure that you added a valid authenticated user for the `admin.users` attribute:
++
+[subs=+quotes]
+----
+catalog:
+      providers:
+        githubOrg:
+          default:
+            id: development
+            orgUrl: ${GITHUB_ORG_URL}
+*permission:
+      enabled: true
+      rbac:
+        admin:
+          users:
+            - name: user:/default/rsriniva*
+----
+
+. Re-deploy the `rhdh` helm chart for the changes to take effect.
+
+. Sign out from the existing RHDH session and login again using the GitHub ID of the declared policy administrator.
+
+. Navigate to the `Catalog` page in RHDH, and click `CREATE`. Note that you are not allowed to create new components.
```

Comment:
so I could reach the create page with the templates overview page. in my case no template has been loaded so the listing is empty but going to create itself works.

Reply:
re-wrote this part. found some bugs in RBAC. Engg knows about this. https://github.com/janus-idp/backstage-showcase/issues/776